docs/src/components/frame/frame.module.css

```diff
@@ -45,3 +45,9 @@
     margin-right: 0;
   }
 }
+
+.example {
+  border-radius: 0.2em;
+  margin: var(--psLayoutSpacingMedium);
+  background: var(--psColorsWhite);
+}
```

Comment:
iframes will always be in light theme. i could pass the prop through the message bus and handling theme toggling...but i don't want too 😭

Reply:
Could read the cookie on the iframe side and use your own Theme.Provider

Reply:
the library i'm using to do the resize also supports the onMessage. i'll spike on it this morning now that i'm fresh and not hating the iframe 😄